.github/workflows/staging.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy Migrations to Staging
+
+on:
+  push:
+    branches:
+      - develop
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
```

Comment:
Please use environments so that we have uniform variable names and env set up in our full infra. See: https://github.com/hypercerts-org/hypercerts-indexer/blob/main/.github/workflows/supabase-ci-staging.yml